lib/rucio/core/replica.py

```diff
@@ -3908,8 +3909,13 @@ def get_suspicious_files(rse_expression, available_elsewhere, filter_=None, logg
     query = query.filter(not_(other_states_present))
 
     # finally, the results are grouped by RSE, scope, name and required to have
-    # at least 'nattempts' occurrences in the result of the query per replica
-    query_result = query.group_by(models.RSEFileAssociation.rse_id, bad_replicas_alias.scope, bad_replicas_alias.name).having(func.count() > nattempts).all()
+    # at least 'nattempts' occurrences in the result of the query per replica.
+    # If nattempts_exact, then only replicas are required to have exactly
+    # 'nattempts' occurences.
+    if nattempts_exact:
+        query_result = query.group_by(models.RSEFileAssociation.rse_id, bad_replicas_alias.scope, bad_replicas_alias.name).having(func.count() == nattempts).all()
```

Comment:
I'm not understanding something here. Wouldn't it be easier to just test for `>=` always? Is there a different way to handle `==` and `>` later on that I'm not seeing?

Reply:
The idea is that we want to create rules for replicas with exactly `nattempts==1`. Whether or not this should be applied to replicas with `nattempts>=1` as well is still up for discussion.

Reply:
:+1: